Make sure music_pending_song is defined before we try using it

http://bugs.gentoo.org/109272

--- pathological.py
+++ pathological.py
@@ -28,6 +28,7 @@
 colorblind = 0
 sound_on = 1
 music_on = 1
+music_pending_song = 0
 for arg in sys.argv[1:]:
 	if arg == '-s':
 		screenshot = 1
@@ -194,7 +195,7 @@
 	sound_on = sound_on ^ 1
 
 def toggle_music():
-	global music_on
+	global music_pending_song, music_on
 	music_on = music_on ^ 1
 	if music_on:
 		if music_pending_song:
